### DISCUSSION TITLE
SSL Cert renewal seems to hang

### DISCUSSION THREAD

**Discussion by ewanc26** [community/community#118718] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi there. I have my [website](https://github.com/ewanc26/website) on Pages (domain from OVH), and from this screenshot, the SSL expired a few days ago.

![Screenshot 2024-04-11 at 18 25 03](https://github.com/community/community/assets/156911937/9f021d8f-bfd4-48ff-9262-c388996ff494)

I thought as I had everything in my DNS set correctly, that it wouldn't be an issue.

Apparently not, as with the title, the renewal seems to hang. I'm not sure what I've done wrong since the DNS check is valid.

![image](https://github.com/community/community/assets/156911937/a0b16e59-0908-41d1-8192-e0b198c0f4cd)

Here is the `dig` output:

```zsh
 ewan@Mac-Mini  ~  dig ewancroft.uk

; <<>> DiG 9.10.6 <<>> ewancroft.uk
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 60794
;; flags: qr rd ra; QUERY: 1, ANSWER: 5, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;ewancroft.uk.			IN	A

;; ANSWER SECTION:
ewancroft.uk.		300	IN	A	185.199.109.153
ewancroft.uk.		300	IN	A	185.199.110.153
ewancroft.uk.		300	IN	A	185.199.108.153
ewancroft.uk.		300	IN	A	185.199.111.153
ewancroft.uk.		300	IN	A	213.186.33.5

;; Query time: 258 msec
;; SERVER: 192.168.1.1#53(192.168.1.1)
;; WHEN: Thu Apr 11 18:28:59 BST 2024
;; MSG SIZE  rcvd: 121
```

I don't know if it's because I've been experimenting with Yunohost on a subdomain, but if it is, should I just wait for longer?

**Comment by yoannchaudet** [comment]
You are having troubles because of this line:

```
ewancroft.uk.		300	IN	A	213.186.33.5
```

Remove that entry from your DNS configuration.

**Reply by ewanc26** [reply]
Thank you! I know it was something so simple but I just was clueless. It's now at cert activation, so that was indeed the issue. Once again, I appreciate the reply.
